test/operators/update-fee.ts

```diff
@@ -0,0 +1,183 @@
+//Declare imports
+import * as helpers from '../helpers/contract-helpers';
+import { expect } from 'chai';
+import { progressTime } from '../helpers/utils';
+import { trackGas, GasGroup } from '../helpers/gas-usage';
+
+//Declare globals
+let ssvNetworkContract: any, initialFee: any;
+
+describe('Operator Fee Tests', () => {
+  beforeEach(async () => {
+    ssvNetworkContract = (await helpers.initializeContract()).contract;
+    initialFee = helpers.CONFIG.minimalOperatorFee * 10;
+    await helpers.registerOperators(2, 1, initialFee);
+  });
+
+  it('Get operator fee', async () => {
+    expect(await ssvNetworkContract.getOperatorFee(1)).to.equal(initialFee);
+  });
+
+  it('Get operator fee that does not exsist reverts "OperatorNotFound"', async () => {
+    await expect(ssvNetworkContract.getOperatorFee(12
+    )).to.be.revertedWith('OperatorNotFound');
+  });
+
+  //Declare
+  it('Declare fee emits "OperatorFeeDeclaration"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[2]).declareOperatorFee(1, initialFee + initialFee / 10
+    )).to.emit(ssvNetworkContract, 'OperatorFeeDeclaration');
+  });
+
+  it('Declare fee in contract', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee - initialFee / 20), [GasGroup.REGISTER_OPERATOR]);
+  });
+
+  it('Declare fee gas limits', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+  });
+
+  it('Declare fee of operator I do not own reverts "CallerNotOwner"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).declareOperatorFee(1, initialFee + initialFee / 10
+    )).to.be.revertedWith('CallerNotOwner');
+  });
+
+  //Test to be removed - Duplicate
+  it('Declare fee of operator I do not own reverts "CallerNotOwner"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).declareOperatorFee(1, initialFee + initialFee / 10
+    )).to.be.revertedWith('CallerNotOwner');
+  });
+
+  it('Declare fee with a wrong Publickey reverts "OperatorWithPublicKeyNotExist"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).declareOperatorFee(12, initialFee + initialFee / 10
+    )).to.be.revertedWith('OperatorWithPublicKeyNotExist');
+  });
+
+  it('Declare fee with fee too low reverts "FeeTooLow"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[2]).declareOperatorFee(1, helpers.CONFIG.minimalOperatorFee - 1
+    )).to.be.revertedWith('FeeTooLow');
+  });
+
+  it('Declare fee is above limit reverts "FeeExceedsIncreaseLimit"', async () => {
```

Comment:
what does limit mean?
Maybe:
Declare fee above the operators max fee increase limit ...
i got "operator max fee increase" because we have a function called updateOperatorFeeIncreaseLimit and just took words from here :)